packages/features/bookings/lib/handleNewRecurringBooking.ts

```diff
@@ -44,7 +44,7 @@ export const handleNewRecurringBooking = async (
       thirdPartyRecurringEventId,
       numSlotsToCheckForAvailability,
       currentRecurringIndex: 0,
-      noEmail: false,
+      noEmail: req.noEmail !== undefined ? req.noEmail : false,
```

Comment:
I think it makes sense to not always have `noEmail: false` if platform emails are disabled.